examples/PSyclone/psyclone.py

```diff
@@ -137,27 +150,29 @@ def options(self, psy):
                         siblings = loop.parent.children
                         my_index = siblings.index(loop)
                         option = []
-                        self._recurse(siblings, my_index, option, my_options, invoke)
+                        self._recurse(siblings, my_index, option, my_options,
+                                      invoke)
 
         return my_options
 
     @property
     def name(self):
-        ''' '''
+        '''Returns the name of the melody input class '''
         return self._name
 
-#FILE = "/home/rupert/proj/GungHo/PSyclone_trunk/examples/gocean/shallow_alg.f90"
-FILE = "/home/rupert/proj/GungHoSVN/PSyclone_trunk/examples/gocean/shallow_alg.f90"
-from parse import parse
-from psyGen import PSyFactory
-_, invoke_info = parse(FILE, api="gocean1.0")
-psy = PSyFactory("gocean1.0").create(invoke_info)
+#FILE = "/home/rupert/proj/GungHo/PSyclone_trunk/examples/gocean/
+#shallow_alg.f90"
+FILE = ("/home/rupert/proj/GungHoSVN/PSyclone_trunk/examples/"
```

Comment:
Code uses a file that is not part of this project. I suggest copying that file into melody/examples.